hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/autoassign/AutoAssignChecker.java

```diff
@@ -100,7 +75,7 @@ public void check() {
 
         final PageRequest pageRequest = PageRequest.of(0, PAGE_SIZE);
 
-        final Page<TargetFilterQuery> filterQueries = targetFilterQueryManagement.findWithAutoAssignDS(pageRequest);
+        final Page<TargetFilterQuery> filterQueries = findWithAutoAssignDS(pageRequest);
```

Comment:
I would extract this to the abstract class following the same logic as in JpaSystemManagement#forEachTenant